### PR TITLE
Statsforecast V4 - Implement describe() method

### PR DESCRIPTION
**Pull request by sammiller06**
## Description

Implements the `describe()` method for the StatsForecast handler.

## Type of change

- [x] ⚡ New feature (non-breaking change which adds functionality)

### What is the solution?

New code and tests in the existing handler files.
Accuracy is based on in-sample cross-validation results.
See the specification for the method at https://docs.google.com/spreadsheets/d/1vm3N5f8IZZmuCyXhgP_-V1ZFDY8aCug8WfyxcQLpl20/edit#gid=0

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] I have shared a short loom video or screenshots demonstrating any new functionality.

### Describe() method for default model (AutoARIMA)
image

### With auto selection - gives separate accuracy for each model
image

### With model attribute
image

### With feature attribute
image

### With ensemble attribute
image

### CODE REVIEW

File: tests/unit/ml_handlers/test_statsforecast.py [hunk 1]
```diff
@@ -152,6 +164,27 @@ def test_model_choice(self, mock_handler):
         assert len(mindsdb_result) == prediction_horizon * n_groups
         assert np.allclose(mindsdb_result, package_predictions)
 
+        # test describe() method
```
Comment (paxcema) [inline tests/unit/ml_handlers/test_statsforecast.py:167]: Just for future maintenance, would be useful to comment the expected data structure for nested fields (in particular, `accuracies` seems tricky to interpret just based on these assertions).

File: tests/unit/ml_handlers/test_statsforecast.py [hunk 2]
```diff
@@ -198,3 +231,8 @@ def test_auto_model_selection(self, mock_handler):
         # check against ground truth
         mindsdb_result = result_df.iloc[:, -1]
         assert np.allclose(mindsdb_result, package_predictions)
+
+        describe_result = self.run_sql("describe proj.modelx")
+        assert len(describe_result["accuracies"][0]) > 1
```
Comment (paxcema) [inline tests/unit/ml_handlers/test_statsforecast.py:236]: Same as in the comment above, would be nice to have a short explanation of the expected data structure.

File: tests/unit/ml_handlers/test_statsforecast.py (hunk 1 above)
Comment (paxcema) [reply tests/unit/ml_handlers/test_statsforecast.py:167]: Okay, debatable if it's truly necessary, going through the actual source of `describe` is enough. Definitely not critical, would be a nice to have.

**Review by paxcema — APPROVED** [review]
Looks good! Just two nitpicks, nothing critical.

File: tests/unit/ml_handlers/test_statsforecast.py (hunk 1 above)
Comment (sammiller06) [reply tests/unit/ml_handlers/test_statsforecast.py:167]: Think you're right, it's not much work to add.

**Pull request merged**